Add datafiller results fix #887

revolunet opened the pull request:
Basic datafiller data integration :

 - passage de field "slug" de `keyword` en `text` pour pouvoir faire des recherches partielles
 - script de fetch/clean données du datafiller
 - intégration dans l'API

Choix qui ont été faits, à discuter :

 - lors de la récupération des données du datafiller, les données sont comparées à notre index ES pour stocker "en dur" les vraies valeurs renvoyées par ES. (et detecter les urls qui ont changées) au lieu de le faire au runtime
 - côté API, on intercepte les queries du front, et on les compare aux recherches gérées par le datafiller et on fournit les résultats comme ES

Dans un 2eme temps je proposerai une PR pour augmenter les recherches qui peuvent utiliser ces données avec un peu de fuzzy

Commit:
cleanup + docs

Commit:
fix(es): make slug field a text field

Commit:
feat(api): include datafiller results

Commit:
doc

lionelB commented on packages/code-du-travail-data/dataset/datafiller/fetch.js:
```diff
@@ -22,48 +23,50 @@ const getSource = url => {
+    (row.variants && row.variants.split("\n").map(variant => variant.trim())) ||
+    [];
+  const variants = [row.title.replace("-", " ")].concat(others);
+  return variants;
```
should you return a `[...new Set(variants)]` to get only unique values ?

revolunet replied:
yep, why not :)

lionelB commented on packages/code-du-travail-api/src/server/routes/search/search.prequalified.js:
```diff
@@ -0,0 +1,40 @@
+      hits: {
+        hits: knownQuery.refs.filter(
+          ref =>
+            !excludeSources.length ||
```
`indexOf`  should be enough ?

revolunet replied:
indeed 👌replaced by `Array.includes`

Commit:
fix review

revolunet commented:
bump

lionelB commented:
@revolunet your branch seems to not be deployed. it would be nice if we could measure it upon https://bnnp9.codesandbox.io/ (which should probably trigger a 100% relevance )

revolunet commented:
Pour une raison inconnue, j'ai cette erreur dans la CI : 
côté API, il ne retrouve pas `require("@cdt/data...datafiller/prequalified.json");`

lionelB reviewed (commented):
@julien  you will need to add 
```dockerfile
COPY ./packages/code-du-travail-data/dataset/datafiller/package.json /app/packages/code-du-travail-data/dataset/datafiller/package.json 
```
in the main Dockerfile

Commit:
wth

Commit:
fix: snapshot

Commit:
lintfix

Commit:
snapshots

lionelB commented:
@revolunet if you rebase your PR, since ES 6.8 is now merge, snapshot PB should be fix

revolunet commented:
bump #887

revolunet commented:
je vais en profiter pour corriger l'url de l'API dans datafiller + corriger les 39 liens HS

Commit:
data: update prequalified.json

revolunet commented:
trop bien le [nouvel outil de mesure](https://bnnp9.codesandbox.io/) :)

<img width="649" alt="Capture d’écran 2019-06-25 à 14 43 06" src="https://user-images.githubusercontent.com/124937/60099271-99375500-9757-11e9-868d-d2807bc049ae.png">

Commit:
update snapshots

lionelB commented:
@julien yeah! Reste à savoir pourquoi toi n'est pas vert 🤔

UnbearableBear commented:
Eh beh, l'outil de mesure est top !!

julien commented:
@lionelB effectivement tout n'est pas en vert

lionelB commented:
@julien 😭 oups pardon pour le poke

Commit:
update prequalified.json

Commit:
update snapshots

revolunet commented:
J'ai fait une petite analyse des urls renvoyées par datafiller. Comme les urls de nos contenus changent, le script d'import tente de mettre à jour selon les urls connues dans ES. en exigeant une similarité > 70%, il reste 14 références interne non trouvées sur 1662 urls référencées. 173 sont corrigées.

Le reste (51) sont des urls externes

Détail : 


status | query | slug | found
------|------|------|------
ERROR | ai je droit à l'allocation chômage en cas de rupture conventionnelle | la-rupture-conventionnelle-du-contrat-de-travail-a-duree-indeterminee-en-quoi-consiste-la-rupture-conventionnelle | not found in ES
ERROR | ai je droit à l'allocation chômage en cas de rupture conventionnelle | la-rupture-conventionnelle-du-contrat-de-travail-a-duree-indeterminee-quelles-sont-les-indemnites-dues-au-salarie | not found in ES
ERROR | calcul indemnité rupture conventionnelle | la-rupture-conventionnelle-du-contrat-de-travail-a-duree-indeterminee-quelles-sont-les-indemnites-dues-au-salarie | not found in ES
ERROR | certificat de travail | les-documents-remis-aux-salaries-lors-de-la-rupture-du-contrat-de-travail-quand-delivrer-le-certificat-de-travail | not found in ES
ERROR | comment-se-faire-licencier | la-rupture-conventionnelle-du-contrat-de-travail-a-duree-indeterminee-en-quoi-consiste-la-rupture-conventionnelle | not found in ES
ERROR | conge-maternite | les-garanties-liees-a-la-maternite-ou-a-ladoption-comment-est-garanti-le-droit-aux-conges-payes | not found in ES
ERROR | conges-payes-decompte | les-garanties-liees-a-la-maternite-ou-a-ladoption-comment-est-garanti-le-droit-aux-conges-payes | not found in ES
ERROR | je n'ai pas signé de contrat de travail travail illégal | les-obligations-de-lemployeur-lors-de-lembauche-quest-ce-que-la-declaration-prealable-a-lembauche | not found in ES
ERROR | remise de mes documents de fin de contrat | les-documents-remis-aux-salaries-lors-de-la-rupture-du-contrat-de-travail-et-lattestation-pour-pole-emploi | not found in ES
ERROR | rupture conventionnelle individuelle indemnité de rupture | la-rupture-conventionnelle-du-contrat-de-travail-a-duree-indeterminee-quelles-sont-les-indemnites-dues-au-salarie | not found in ES
ERROR | rupture-conventionnelle | la-rupture-conventionnelle-du-contrat-de-travail-a-duree-indeterminee-en-quoi-consiste-la-rupture-conventionnelle | not found in ES
ERROR | rupture-conventionnelle | la-rupture-conventionnelle-du-contrat-de-travail-a-duree-indeterminee-les-salaries-proteges-sont-ils-concernes | not found in ES
ERROR | rupture-conventionnelle | la-rupture-conventionnelle-du-contrat-de-travail-a-duree-indeterminee-quelles-sont-les-indemnites-dues-au-salarie | not found in ES
ERROR | tickets-restaurant | quelles-sont-les-conditions-dobtention-et-dutilisation-des-titres-restaurant | not found in ES
EXTERNAL | calcul indemnité rupture conventionnelle | https://www.telerc.travail.gouv.fr/RuptureConventionnellePortailPublic/jsp/site/Portal.jsp?page_id=14 | external link
EXTERNAL | cheque vacances | https://www.service-public.fr/particuliers/vosdroits/F2309 | external link
EXTERNAL | chsct | https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006072050&idArticle=LEGIARTI000031085511&dateTexte=&categorieLien=id | external link
EXTERNAL | combien est on payé en accident de travail | https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=DEB613CACFA6CCD8A7EBAB4F40901326.tplgfr27s_1?idArticle=LEGIARTI000006743045&cidTexte=LEGITEXT000006073189&dateTexte=20190418&categorieLien=id&oldAction=&nbResultRech= | external link
EXTERNAL | combien est on payé en accident de travail | https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=DEB613CACFA6CCD8A7EBAB4F40901326.tplgfr27s_1?idArticle=LEGIARTI000006750304&cidTexte=LEGITEXT000006073189&dateTexte=20190418 | external link
EXTERNAL | combien est on payé en accident de travail | https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=DEB613CACFA6CCD8A7EBAB4F40901326.tplgfr27s_1?idArticle=LEGIARTI000006750307&cidTexte=LEGITEXT000006073189&dateTexte=20190418&categorieLien=id&oldAction=&nbResultRech= | external link
EXTERNAL | combien est on payé en accident de travail | https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=DEB613CACFA6CCD8A7EBAB4F40901326.tplgfr27s_1?idArticle=LEGIARTI000006750310&cidTexte=LEGITEXT000006073189&dateTexte=20190418&categorieLien=id&oldAction=&nbResultRech= | external link
EXTERNAL | combien est on payé en accident de travail | https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=DEB613CACFA6CCD8A7EBAB4F40901326.tplgfr27s_1?idArticle=LEGIARTI000025014025&cidTexte=LEGITEXT000006073189&dateTexte=20190418 | external link
EXTERNAL | combien est on payé en accident de travail | https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=DEB613CACFA6CCD8A7EBAB4F40901326.tplgfr27s_1?idArticle=LEGIARTI000029396214&cidTexte=LEGITEXT000006073189&dateTexte=20190418&categorieLien=id&oldAction=&nbResultRech= | external link
EXTERNAL | comment rompre mon contrat d'apprentissage | https://travail-emploi.gouv.fr/formation-professionnelle/formation-en-alternance-10751/contrat-apprentissage | external link
EXTERNAL | complement-salaire pole emploi | https://www.service-public.fr/particuliers/vosdroits/F14860 | external link
EXTERNAL | compte personnel de formation | https://travail-emploi.gouv.fr/formation-professionnelle/droits-a-la-formation-et-orientation/compte-personnel-formation | external link
EXTERNAL | compte personnel de formation | https://www.moncompteactivite.gouv.fr/cpa-public/ | external link
EXTERNAL | conges-assistante-maternelle | https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=824878D1C70B699E273444045FD6A090.tplgfr22s_3?idArticle=LEGIARTI000006798032&cidTexte=LEGITEXT000006074069&dateTexte=20190424 | external link
EXTERNAL | conges-assistante-maternelle | https://www.legifrance.gouv.fr/affichIDCC.do;jsessionid=CB38A4CCACBD9F4F8EC46CDB36FAD5B8.tplgfr22s_3?idSectionTA=KALISCTA000005713212&cidTexte=KALITEXT000005669667&idConvention=KALICONT000005635807 | external link
EXTERNAL | contrat de professionnalisation | https://travail-emploi.gouv.fr/publications/picts/bo/30082012/TRE_20120008_0110_0005.pdf | external link
EXTERNAL | contrat de professionnalisation | https://www.alternance.emploi.gouv.fr/portail_alternance/jcms/recleader_6113/decouvrir-l-alternance | external link
EXTERNAL | contrat-apprentissage | https://www.alternance.emploi.gouv.fr/portail_alternance/ | external link
EXTERNAL | cumul emploi retraite | https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=8F5B457AE071BC036886DFCFA271726B.tplgfr44s_2?idArticle=LEGIARTI000028496884&cidTexte=LEGITEXT000006073189&dateTexte=20190607 | external link
EXTERNAL | cumul emploi retraite | https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=8F5B457AE071BC036886DFCFA271726B.tplgfr44s_2?idArticle=LEGIARTI000033713940&cidTexte=LEGITEXT000006073189&dateTexte=20190607&categorieLien=id&oldAction= | external link
EXTERNAL | demission chomage | https://www.legifrance.gouv.fr/affichCode.do?idSectionTA=LEGISCTA000037388242&cidTexte=LEGITEXT000006072050 | external link
EXTERNAL | demission-rapprochement-conjoint | https://www.unedic.org/indemnisation/textes-reglementaires/conventions-dassurance-chomage/accord-dapplication-ndeg-14-du | external link
EXTERNAL | différencier salarié, indépendant, et faux indépendant? | https://www.service-public.fr/professionnels-entreprises/vosdroits/F1691 | external link
EXTERNAL | dpae | https://www.urssaf.fr/portail/home/les-risques-du-travail-dissimule/les-risques-du-travail-dissimule/salarie-comment-savoir-si-vous-e.html | external link
EXTERNAL | droits-chomage | https://www.service-public.fr/particuliers/vosdroits/F14860 | external link
EXTERNAL | droits-chomage | https://www.service-public.fr/particuliers/vosdroits/F2064 | external link
EXTERNAL | epargne salariale | https://travail-emploi.gouv.fr/droit-du-travail/l-epargne-salariale/article/epargne-salariale-principes-generaux | external link
EXTERNAL | epargne salariale | https://travail-emploi.gouv.fr/droit-du-travail/l-epargne-salariale/article/les-avantages-sociaux-et-fiscaux-de-l-epargne-salariale | external link
EXTERNAL | epargne salariale | https://travail-emploi.gouv.fr/droit-du-travail/l-epargne-salariale/article/les-plans-d-epargne-salariale | external link
EXTERNAL | epargne salariale | https://www.service-public.fr/particuliers/vosdroits/F10260 | external link
EXTERNAL | epargne salariale | https://www.service-public.fr/particuliers/vosdroits/F2141 | external link
EXTERNAL | epargne salariale | https://www.service-public.fr/particuliers/vosdroits/N517 | external link
EXTERNAL | formulaire accident du travail | https://www.service-public.fr/particuliers/vosdroits/R14587 | external link
EXTERNAL | formulaire accident du travail | https://www.service-public.fr/particuliers/vosdroits/R19231 | external link
EXTERNAL | intéressement participation  | https://travail-emploi.gouv.fr/droit-du-travail/l-epargne-salariale/article/l-interessement-et-la-participation | external link
EXTERNAL | intéressement participation  | https://www.service-public.fr/particuliers/vosdroits/F2141 | external link
EXTERNAL | licenciement-economique | https://www.service-public.fr/particuliers/vosdroits/N481 | external link
EXTERNAL | modification-lieu-travail | https://www.service-public.fr/particuliers/vosdroits/F2339 | external link
EXTERNAL | mutuelle | https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006073189&idArticle=LEGIARTI000027549125&dateTexte=&categorieLien=cid | external link
EXTERNAL | mutuelle | https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000027546648 | external link
EXTERNAL | Parcours emploi compétences | https://travail-emploi.gouv.fr/emploi/parcours-emploi-competences/article/parcours-emploi-competences | external link
EXTERNAL | plan d'épargne salariale | https://travail-emploi.gouv.fr/droit-du-travail/l-epargne-salariale/article/epargne-salariale-principes-generaux | external link
EXTERNAL | plan d'épargne salariale | https://travail-emploi.gouv.fr/droit-du-travail/l-epargne-salariale/article/les-plans-d-epargne-salariale | external link
EXTERNAL | plan d'épargne salariale | https://travail-emploi.gouv.fr/ministere/documentation-et-publications-officielles/guides/article/guide-de-l-epargne-salariale | external link
EXTERNAL | prime-precarite | https://codedutravail-dev.num.social.gouv.fr/fiche-ministere-travail/cdd-a-objet-defini-ou-cdd-de-mission-une-indemnite-de-fin-de-contrat-est-elle-due?q=indemnit%C3%A9%20de%20fin%20de%20contrat%20CDD | external link
EXTERNAL | prime-precarite | https://www.service-public.fr/particuliers/vosdroits/F40 | external link
EXTERNAL | rupture conventionnelle collective | https://travail-emploi.gouv.fr/ministere/documentation-et-publications-officielles/guides/QR-RCC | external link
EXTERNAL | rupture conventionnelle individuelle indemnité de rupture | https://www.telerc.travail.gouv.fr/RuptureConventionnellePortailPublic/jsp/site/Portal.jsp?page_id=14 | external link
EXTERNAL | rupture conventionnelle individuelle | https://www.telerc.travail.gouv.fr/RuptureConventionnellePortailPublic/jsp/site/Portal.jsp?page_id=1 | external link
EXTERNAL | rupture-assistance-maternelle | https://www.legifrance.gouv.fr/affichIDCC.do;jsessionid=3464C2C39FEB50A29E251F50C9139E3F.tplgfr27s_1?idSectionTA=KALISCTA000005713278&cidTexte=KALITEXT000005669667&idConvention=KALICONT000005635807 | external link
EXTERNAL | rupture-assistance-maternelle | https://www.legifrance.gouv.fr/affichIDCC.do?cidTexte=KALITEXT000005669667&idSectionTA=KALISCTA000005713182&idConvention=KALICONT000005635807&dateTexte=29990101 | external link
EXTERNAL | travail-illegal | https://www.urssaf.fr/portail/home/les-risques-du-travail-dissimule.html | external link
FIXED |  au secours | le-harcelement-moral-quels-recours | harcelement-moral-quels-recours
FIXED |  au secours | le-harcelement-moral-qui-organise-la-prevention-en-matiere-de-harcelement-moral | harcelement-moral-qui-organise-la-prevention-en-matiere-de-harcelement-moral
FIXED |  au secours | linspection-du-travail-quelles-sont-les-missions | inspection-du-travail-quelles-sont-les-missions
FIXED | 13eme-mois | le-salaire-comment-est-fixe-le-salaire-de-base | salaire-comment-est-fixe-le-salaire-de-base
FIXED | 13eme-mois | le-salaire-quelles-sont-les-modalites-de-paiement | salaire-quelles-sont-les-modalites-de-paiement
FIXED | 13eme-mois | pour-quels-motifs-un-accord-de-performance-collective-peut-il-etre-negocie-dans-lentreprise-zooygwjdbcv4ig | pour-quels-motifs-un-accord-de-performance-collective-peut-il-etre-negocie-dans-lentreprise-rmlonwxpjk95bw
FIXED | A quelle date l'employeur doit il régler mon salaire | le-salaire-quelles-sont-les-modalites-de-paiement | salaire-quelles-sont-les-modalites-de-paiement
FIXED | abandon-de-poste | labandon-de-poste-est-il-une-demission | demission-le-refus-par-le-salarie-daccepter-un-changement-des-conditions-de-travail-est-il-une-demission
FIXED | absence-contrat-travail | cdd-dans-quels-cas-le-contrat-a-duree-determinee-peut-il-etre-requalifie-en-contrat-a-duree-indeterminee | cdd-ce-que-dit-le-code-du-travail-dans-quels-cas-le-contrat-a-duree-determinee-peut-il-etre-requalifie-en-contrat-a-duree-indeterminee
FIXED | absence-contrat-travail | le-contrat-de-travail-temporaire-quelle-est-la-duree-du-contrat | contrat-de-travail-temporaire-quelle-est-la-duree-du-contrat
FIXED | accident-travail | larret-de-travail-pour-accident-du-travail-ou-maladie-professionnelle-que-se-passe-t-il-pendant-larret-de-travail | arret-de-travail-pour-accident-ou-maladie-pro-que-se-passe-t-il-pendant-larret-de-travail
FIXED | ai je droit à l'allocation chômage en cas de rupture conventionnelle | la-rupture-conventionnelle-collective-comment-la-rupture-conventionnelle-collective-est-elle-mise-en-oeuvre | rupture-conventionnelle-collective-comment-la-rupture-conventionnelle-collective-est-elle-mise-en-oeuvre
FIXED | allocation-chomage | demandeur-demploi-createur-ou-repreneur-dentreprise-quels-sont-les-dispositifs-pour-les-beneficiaires-de-lallocation-assurance-chomage-are | cumul-des-allocations-et-des-revenus-dactivite-quels-sont-les-dispositifs-pour-les-beneficiaires-de-lallocation-assurance-chomage-are
FIXED | allocation-chomage | le-droit-aux-allocations-chomage-du-salarie-demissionnaire-que-se-passe-t-il-dans-les-autres-cas | droit-aux-allocations-chomage-en-cas-de-demission-que-se-passe-t-il-dans-les-autres-cas
FIXED | allocation-chomage | le-droit-aux-allocations-chomage-du-salarie-demissionnaire-que-se-passe-t-il-dans-les-autres-cas | droit-aux-allocations-chomage-en-cas-de-demission-que-se-passe-t-il-dans-les-autres-cas
FIXED | arret-travail | larret-de-travail-pour-accident-du-travail-ou-maladie-professionnelle-que-se-passe-t-il-pendant-larret-de-travail | arret-de-travail-pour-accident-ou-maladie-pro-que-se-passe-t-il-pendant-larret-de-travail
FIXED | assistance entretien préalable | la-procedure-en-cas-de-licenciement-individuel-pour-motif-economique-en-quoi-consiste-lentretien-prealable-au-licenciement | licenciement-pour-motif-economique-en-quoi-consiste-lentretien-prealable-au-licenciement
FIXED | assistance entretien préalable | la-procedure-en-cas-de-licenciement-pour-motif-personnel-en-quoi-consiste-lentretien-prealable | licenciement-pour-motif-personnel-en-quoi-consiste-lentretien-prealable
FIXED | assistance entretien préalable | le-conseiller-du-salarie-comment-se-faire-assister-par-un-conseiller-du-salarie | entretien-prealable-de-licenciement-comment-se-faire-assister-par-un-conseiller-du-salarie
FIXED | assistance entretien préalable | le-conseiller-du-salarie-quel-est-le-champ-dintervention-du-conseiller-du-salarie | entretien-prealable-de-licenciement-quel-est-le-champ-dintervention-du-conseiller-du-salarie
FIXED | assistance entretien préalable | le-conseiller-du-salarie-qui-peut-etre-conseiller-du-salarie | 164-le-conseiller-du-salarie
FIXED | avenant-contrat-travail | la-mobilite-volontaire-securisee-faut-il-etablir-un-avenant-au-contrat-de-travail | mobilite-volontaire-securisee-faut-il-etablir-un-avenant-au-contrat-de-travail
FIXED | avenant-contrat-travail | la-modification-du-contrat-de-travail-le-salarie-dispose-t-il-dun-delai-pour-refuser-la-modification-du-contrat | modification-du-contrat-de-travail-le-salarie-dispose-t-il-dun-delai-pour-refuser-la-modification-du-contrat
FIXED | avenant-contrat-travail | la-modification-du-contrat-de-travail-lemployeur-peut-il-modifier-le-contrat-de-travail | modification-du-contrat-de-travail-lemployeur-peut-il-modifier-le-contrat-de-travail
FIXED | avenant-contrat-travail | la-modification-du-contrat-de-travail-quelles-sont-les-consequences-dun-refus-du-salarie-ou-dun-defaut-de-reponse | modification-du-contrat-de-travail-quelles-sont-les-consequences-dun-refus-du-salarie-ou-dun-defaut-de-reponse
FIXED | avenant-contrat-travail | le-travail-a-temps-partiel-en-quoi-consistent-les-complements-dheures-par-avenant | contrat-de-travail-a-temps-partiel-en-quoi-consistent-les-complements-dheures-par-avenant
FIXED | calcul indemnité rupture conventionnelle | la-rupture-conventionnelle-du-contrat-de-travail-a-duree-indeterminee-quel-est-le-contenu-de-la-convention | rupture-conventionnelle-quel-est-le-contenu-de-la-convention
FIXED | calcul-anciennete | le-conge-parental-deducation-que-se-passe-t-il-a-lissue-du-conge | conge-parental-deducation-que-se-passe-t-il-a-lissue-du-conge
FIXED | calcul-conges-payes | les-conges-payes-comment-calculer-le-nombre-de-jours-de-conges | conges-payes-comment-calculer-le-nombre-de-jours-de-conges
FIXED | calcul-conges-payes | les-conges-payes-quelles-sont-les-consequences-de-la-maladie-durant-les-conges-payes | conges-payes-quelles-sont-les-consequences-de-la-maladie-durant-les-conges-payes
FIXED | calcul-conges-payes | les-conges-payes-quelles-sont-les-modalites-de-prise-des-conges-payes | conges-payes-quelles-sont-les-modalites-de-prise-des-conges-payes
FIXED | calcul-conges-payes | les-conges-payes-qui-a-droit-aux-conges-payes | conges-payes-qui-a-droit-aux-conges-payes
FIXED | calcul-indemnites-licenciement | lindemnite-legale-de-licenciement-comment-calculer-le-salaire-de-reference | indemnite-legale-de-licenciement-comment-calculer-le-salaire-de-reference
FIXED | calcul-indemnites-licenciement | lindemnite-legale-de-licenciement-comment-determiner-le-montant-de-lindemnite | indemnite-legale-de-licenciement-comment-determiner-le-montant-de-lindemnite
FIXED | calcul-solde-de-tout-compte | a-quel-moment-mon-employeur-doit-il-me-fournir-mon-solde-tout-compte-jq-udksinsdgfq | a-quel-moment-mon-employeur-doit-il-me-fournir-mon-solde-tout-compte-sxy2g7jtti3h5a
FIXED | calcul-solde-de-tout-compte | les-documents-remis-aux-salaries-lors-de-la-rupture-du-contrat-de-travail-en-quoi-consiste-le-solde-de-tout-compte | rupture-du-contrat-de-travail-en-quoi-consiste-le-solde-de-tout-compte
FIXED | cdd-vers-cdi | cdd-dans-quels-cas-le-contrat-a-duree-determinee-peut-il-etre-requalifie-en-contrat-a-duree-indeterminee | cdd-ce-que-dit-le-code-du-travail-dans-quels-cas-le-contrat-a-duree-determinee-peut-il-etre-requalifie-en-contrat-a-duree-indeterminee
FIXED | cdd-vers-cdi | cdd-lembauche-sous-contrat-a-duree-determinee | contrat-a-duree-determinee-cdd
FIXED | cdd-vers-cdi | la-remuneration-du-salarie-en-contrat-a-duree-determinee-le-salarie-a-t-il-droit-a-une-indemnite-de-fin-de-contrat | remuneration-du-salarie-en-contrat-a-duree-determinee-le-salarie-a-t-il-droit-a-une-indemnite-de-fin-de-contrat
FIXED | CDI | cdd-dans-quels-cas-le-contrat-a-duree-determinee-peut-il-etre-requalifie-en-contrat-a-duree-indeterminee | cdd-ce-que-dit-le-code-du-travail-dans-quels-cas-le-contrat-a-duree-determinee-peut-il-etre-requalifie-en-contrat-a-duree-indeterminee
FIXED | CDI | comment-le-cdi-peut-il-prendre-fin | cdi-ce-que-dit-le-code-du-travail-comment-le-cdi-peut-il-prendre-fin
FIXED | CDI | dans-quels-cas-un-cdi-doit-il-etre-conclu | cdi-ce-que-dit-le-code-du-travail-dans-quels-cas-un-cdi-doit-il-etre-conclu
FIXED | CDI | dans-quels-cas-un-cdi-doit-il-etre-conclu | cdi-ce-que-dit-le-code-du-travail-dans-quels-cas-un-cdi-doit-il-etre-conclu
FIXED | CDI | le-contrat-a-duree-indeterminee-de-chantier-ou-doperation-quest-ce-quun-contrat-de-chantier-ou-doperation | contrat-de-chantier-ou-doperation-quest-ce-quun-contrat-de-chantier-ou-doperation
FIXED | CDI | quel-est-le-contenu-du-cdi | cdi-ce-que-dit-le-code-du-travail-quel-est-le-contenu-du-cdi
FIXED | CDI | quelle-forme-doit-revetir-le-cdi | cdi-ce-que-dit-le-code-du-travail-quelle-forme-doit-revetir-le-cdi
FIXED | changement-horaires | la-modification-du-contrat-de-travail-lemployeur-peut-il-modifier-le-contrat-de-travail | modification-du-contrat-de-travail-lemployeur-peut-il-modifier-le-contrat-de-travail
FIXED | changement-horaires | la-modification-du-contrat-de-travail-quelles-sont-les-consequences-dun-refus-du-salarie-ou-dun-defaut-de-reponse | modification-du-contrat-de-travail-quelles-sont-les-consequences-dun-refus-du-salarie-ou-dun-defaut-de-reponse
FIXED | chsct | quand-le-chsct-est-il-consulte | hygiene-et-securite-quand-faut-il-consulter-le-chsct-ou-le-cse
FIXED | clause-non-concurrence | la-demission-quelle-est-la-situation-du-salarie-a-la-fin-du-contrat | demission-quelle-est-la-situation-du-salarie-a-la-fin-du-contrat
FIXED | clause-non-concurrence | quel-est-le-contenu-du-cdi | cdi-ce-que-dit-le-code-du-travail-quel-est-le-contenu-du-cdi
FIXED | combien est on payé en accident de travail | larret-de-travail-pour-accident-du-travail-ou-maladie-professionnelle-que-se-passe-t-il-pendant-larret-de-travail | arret-de-travail-pour-accident-ou-maladie-pro-que-se-passe-t-il-pendant-larret-de-travail
FIXED | comment contester mon licenciement  | le-conseil-de-prudhommes-a-quel-conseil-de-prudhommes-sadresser | prudhommes-a-quel-conseil-de-prudhommes-sadresser
FIXED | comment contester mon licenciement  | le-conseil-de-prudhommes-peut-on-se-faire-representer-ou-assister-a-laudience | prudhommes-peut-on-se-faire-representer-ou-assister-a-laudience
FIXED | comment contester mon licenciement  | le-licenciement-pour-motif-personnel-quest-ce-quun-motif-personnel | licenciement-pour-motif-personnel-quest-ce-quun-motif-personnel
FIXED | comment-demissionner | la-demission-comment-presenter-une-demission | demission-comment-presenter-une-demission
FIXED | comment-demissionner | la-demission-et-les-heures-pour-recherche-demploi | demission-et-les-heures-pour-recherche-demploi
FIXED | comment-demissionner | la-demission-faut-il-respecter-un-preavis | demission-faut-il-respecter-un-preavis
FIXED | comment-demissionner | la-demission-le-refus-par-le-salarie-daccepter-un-changement-des-conditions-de-travail-est-il-une-demission | demission-le-refus-par-le-salarie-daccepter-un-changement-des-conditions-de-travail-est-il-une-demission
FIXED | comment-demissionner | la-demission-le-salarie-peut-il-demissionner-dans-le-cadre-dun-contrat-a-duree-determinee | demission-le-salarie-peut-il-demissionner-dans-le-cadre-dun-contrat-a-duree-determinee
FIXED | comment-demissionner-cdd | cdd-dans-quelles-conditions-le-contrat-peut-il-etre-rompu | cdd-ce-que-dit-le-code-du-travail-dans-quelles-conditions-le-contrat-peut-il-etre-rompu
FIXED | comment-demissionner-cdd | la-demission-le-salarie-peut-il-demissionner-dans-le-cadre-dun-contrat-a-duree-determinee | demission-le-salarie-peut-il-demissionner-dans-le-cadre-dun-contrat-a-duree-determinee
FIXED | conge-maternite | conge-de-maternite-dune-salariee-du-secteur-prive | conge-maternite-dune-salariee-du-secteur-prive
FIXED | conge-maternite | le-conge-de-maternite-quelle-est-la-duree-du-conge-de-maternite | conge-de-maternite-quelle-est-la-duree-du-conge-de-maternite
FIXED | conge-maternite | le-conge-de-maternite-quelle-est-la-situation-a-lissue-du-conge | conge-de-maternite-quelle-est-la-situation-a-lissue-du-conge
FIXED | conge-maternite | le-conge-de-maternite-quelles-sont-les-dispositions-specifiques | conge-de-maternite-quelles-sont-les-dispositions-specifiques
FIXED | conge-parental | le-conge-parental-deducation-comment-prendre-le-conge-parental | conge-parental-deducation-comment-prendre-le-conge-parental
FIXED | conge-parental | le-conge-parental-deducation-est-il-possible-dinterrompre-ou-de-modifier-le-conge-parental | conge-parental-deducation-est-il-possible-dinterrompre-ou-de-modifier-le-conge-parental
FIXED | conge-parental | le-conge-parental-deducation-le-conge-parental-peut-il-etre-refuse | conge-parental-deducation-le-conge-parental-peut-il-etre-refuse
FIXED | conge-parental | le-conge-parental-deducation-ouvre-t-il-systematiquement-droit-a-la-prestation-partagee-deducation-de-lenfant-prepare-ou-complement-de-libre-choix-dactivite | conge-parental-deducation-le-conge-parental-deducation-ouvre-t-il-systematiquement-droit-a-la-prestation-partagee-deducation-de-lenfant-prepare-ou-complement-de-libre-choix-dactivite
FIXED | conge-parental | le-conge-parental-deducation-peut-on-exercer-une-activite-professionnelle-suivre-une-formation-durant-le-conge-parental | conge-parental-deducation-peut-on-exercer-une-activite-professionnelle-suivre-une-formation-durant-le-conge-parental
FIXED | conge-parental | le-conge-parental-deducation-que-se-passe-t-il-a-lissue-du-conge | conge-parental-deducation-que-se-passe-t-il-a-lissue-du-conge
FIXED | conge-parental | le-conge-parental-deducation-quelle-est-la-duree-du-conge-parental | conge-parental-deducation-quelle-est-la-duree-du-conge-parental
FIXED | conge-parental | le-conge-parental-deducation-quelles-sont-les-formalites | conge-parental-deducation-quelles-sont-les-formalites
FIXED | conge-parental | le-conge-parental-deducation-quels-salaries | conge-parental-deducation-quels-salaries
FIXED | conge-sabbatique | le-conge-sabbatique-comment-seffectue-la-demande | conge-sabbatique-comment-seffectue-la-demande
FIXED | conge-sabbatique | le-conge-sabbatique-quelle-en-est-la-duree-et-quels-sont-les-effets-sur-le-contrat-de-travail | conge-sabbatique-quelle-en-est-la-duree-et-quels-sont-les-effets-sur-le-contrat-de-travail
FIXED | conge-sabbatique | le-conge-sabbatique-quelle-est-la-situation-du-salarie-a-lissue-du-conge | conge-sabbatique-quelle-est-la-situation-du-salarie-a-lissue-du-conge
FIXED | conge-sabbatique | le-conge-sabbatique-quelle-peut-etre-la-reponse-de-lemployeur | conge-sabbatique-quelle-peut-etre-la-reponse-de-lemployeur
FIXED | conge-sabbatique | le-conge-sabbatique-quels-salaries-y-ont-droit | conge-sabbatique-quels-salaries-y-ont-droit
FIXED | conge-sans-solde | le-conge-sans-solde-la-remuneration-est-elle-maintenue-pendant-le-conge | conge-sans-solde-la-remuneration-est-elle-maintenue-pendant-le-conge
FIXED | conge-sans-solde | le-conge-sans-solde-quelle-est-la-procedure | conge-sans-solde-quelle-est-la-procedure
FIXED | conge-sans-solde | le-conge-sans-solde-quelles-sont-les-caracteristiques | conge-sans-solde-quelles-sont-les-caracteristiques
FIXED | conges-imposes | les-conges-payes-a-quel-moment-prendre-les-conges-payes | conges-payes-a-quel-moment-prendre-les-conges-payes
FIXED | conges-mariage | les-conges-pour-evenements-familiaux-quelle-est-la-duree-du-conge | conges-pour-evenements-familiaux-quelle-est-la-duree-du-conge
FIXED | conges-mariage | les-conges-pour-evenements-familiaux-quelle-est-la-procedure | conges-pour-evenements-familiaux-quelle-est-la-procedure
FIXED | conges-mariage | les-conges-pour-evenements-familiaux-quels-sont-les-evenements-ouvrant-droit-a-conge | conges-pour-evenements-familiaux-quels-sont-les-evenements-ouvrant-droit-a-conge
FIXED | conges-payes | les-conges-payes-comment-calculer-le-nombre-de-jours-de-conges | conges-payes-comment-calculer-le-nombre-de-jours-de-conges
FIXED | conges-payes | les-conges-payes-quappelle-t-on-periode-de-reference | conges-payes-quappelle-t-on-periode-de-reference
FIXED | conges-payes | les-conges-payes-quelle-est-lincidence-de-labsence-pour-maladie-sur-le-droit-aux-conges-payes | conges-payes-quelle-est-lincidence-de-labsence-pour-maladie-sur-le-droit-aux-conges-payes
FIXED | conges-payes | les-conges-payes-quelles-sont-les-modalites-de-prise-des-conges-payes | conges-payes-quelles-sont-les-modalites-de-prise-des-conges-payes
FIXED | conges-payes | les-conges-payes-qui-a-droit-aux-conges-payes | conges-payes-qui-a-droit-aux-conges-payes
FIXED | conges-payes | les-jours-feries-et-les-ponts-que-se-passe-t-il-si-un-jour-ferie-tombe-pendant-les-conges-payes | jours-feries-et-ponts-que-se-passe-t-il-si-un-jour-ferie-tombe-pendant-les-conges-payes
FIXED | conges-payes-decompte | les-conges-payes-comment-calculer-le-nombre-de-jours-de-conges | conges-payes-comment-calculer-le-nombre-de-jours-de-conges
FIXED | conges-payes-decompte | les-conges-payes-jour-ouvrable-de-quoi-sagit-il | conges-payes-jour-ouvrable-de-quoi-sagit-il
FIXED | conges-temps-partiel | le-travail-a-temps-partiel-quels-sont-les-droits-des-salaries-a-temps-partiel | contrat-de-travail-a-temps-partiel-quels-sont-les-droits-des-salaries-a-temps-partiel
FIXED | congés payés imposés | les-conges-payes-a-quel-moment-prendre-les-conges-payes | conges-payes-a-quel-moment-prendre-les-conges-payes
FIXED | congés payés imposés | les-conges-payes-quelles-sont-les-modalites-de-prise-des-conges-payes | conges-payes-quelles-sont-les-modalites-de-prise-des-conges-payes
FIXED | contrat de professionnalisation | contrat-de-professionnalisation-ou-apprentissage-quelles-differences | contrat-de-professionnalisation-et-apprentissage-quelles-differences
FIXED | contrat de professionnalisation | le-contrat-de-professionnalisation-quelles-demarches-pour-formaliser-un-contrat-de-professionnalisation | contrat-de-professionnalisation-alternance-quelles-demarches-pour-formaliser-un-contrat-de-professionnalisation
FIXED | contrat de professionnalisation | le-contrat-de-professionnalisation-quels-contrats-et-conditions-de-travail-du-contrat-de-professionnalisation | contrat-de-professionnalisation-alternance-quels-contrats-et-conditions-de-travail-du-contrat-de-professionnalisation
FIXED | harcèlement moral | article-l1152-1 | l1152-1
FIXED | harcèlement moral | le-harcelement-moral-quelles-sanctions-a-lencontre-de-lauteur-de-harcelement-moral | harcelement-moral-quelles-sanctions-a-lencontre-de-lauteur-de-harcelement-moral
FIXED | harcèlement moral | le-harcelement-moral-quels-recours | harcelement-moral-quels-recours
FIXED | harcèlement moral | le-harcelement-moral-qui-organise-la-prevention-en-matiere-de-harcelement-moral | harcelement-moral-qui-organise-la-prevention-en-matiere-de-harcelement-moral
FIXED | inspection du travail | linspection-du-travail-quels-sont-les-droits-et-obligations-des-agents-de-linspection-du-travail | inspection-du-travail-quels-sont-les-droits-et-obligations-des-agents-de-linspection-du-travail
FIXED | je travaille dans le bâtiment, et mon chef ne relève pas mes heures de travail | article-d3171-8 | d3171-8
FIXED | journee-solidarite | la-journee-de-solidarite-comment-ses-modalites-daccomplissement-sont-elles-fixees | journee-de-solidarite-comment-ses-modalites-daccomplissement-sont-elles-fixees
FIXED | journee-solidarite | la-journee-de-solidarite-peut-elle-etre-fractionnee | journee-de-solidarite-la-journee-de-solidarite-peut-elle-etre-fractionnee
FIXED | journee-solidarite | la-journee-de-solidarite-quelles-en-sont-les-consequences-en-matiere-de-remuneration | journee-de-solidarite-quelles-en-sont-les-consequences-en-matiere-de-remuneration
FIXED | journee-solidarite | la-journee-de-solidarite-quest-ce-que-la-journee-de-solidarite | journee-de-solidarite-quest-ce-que-la-journee-de-solidarite
FIXED | licenciement économique conditions | licenciement-economique-les-obligations-de-lemployeu | licenciement-economique-les-obligations-de-lemployeur
FIXED | licenciement-absences | rupture-de-contrat/le-licenciement-collectif | rupture-du-contrat-de-travail-quand-delivrer-le-certificat-de-travail
FIXED | licenciement-inaptitude | rupture-de-contrat/le-licenciement-collectif | rupture-du-contrat-de-travail-quand-delivrer-le-certificat-de-travail
FIXED | licenciement-procedure | rupture-de-contrat/le-licenciement-collectif | rupture-du-contrat-de-travail-quand-delivrer-le-certificat-de-travail
FIXED | modification du contrat de travail, réduction d'horaires | la-modification-du-contrat-de-travail-la-reduction-du-temps-de-travail-par-voie-daccord-constitue-t-elle-une-modification-du-contrat-de-travail | modification-du-contrat-de-travail-la-reduction-du-temps-de-travail-par-voie-daccord-constitue-t-elle-une-modification-du-contrat-de-travail
FIXED | prudhommes | employeur-comment-etre-rembourse-de-la-remuneration-dun-salarie-conseiller-prudhomal | comment-se-faire-rembourser-la-remuneration-dun-salarie-conseiller-prudhomal
FIXED | prudhommes | le-conseil-de-prudhommes-a-quel-conseil-de-prudhommes-sadresser | prudhommes-a-quel-conseil-de-prudhommes-sadresser
FIXED | prudhommes | le-conseil-de-prudhommes-comment-se-deroule-une-procedure | prudhommes-comment-se-deroule-une-procedure
FIXED | prudhommes | le-conseil-de-prudhommes-existe-t-il-une-procedure-durgence | prudhommes-existe-t-il-une-procedure-durgence
FIXED | prudhommes | le-conseil-de-prudhommes-peut-on-se-faire-representer-ou-assister-a-laudience | prudhommes-peut-on-se-faire-representer-ou-assister-a-laudience
FIXED | Qui dois-je consulter en cas de harcèlement moral? | la-procedure-en-cas-de-licenciement-pour-motif-personnel-quelles-sont-les-sanctions-encourues | licenciement-pour-motif-personnel-quelles-sont-les-sanctions-encourues
FIXED | Qui dois-je consulter en cas de harcèlement moral? | la-protection-contre-les-discriminations-qui-peut-intervenir-en-cas-de-discrimination | protection-contre-les-discriminations-qui-peut-intervenir-en-cas-de-discrimination
FIXED | Qui dois-je consulter en cas de harcèlement moral? | le-harcelement-moral-la-mediation-dans-quelles-conditions | harcelement-moral-la-mediation-dans-quelles-conditions
FIXED | Qui dois-je consulter en cas de harcèlement moral? | le-harcelement-moral-quelle-est-letendue-de-la-protection-des-victimes-et-des-temoins-du-harcelement-moral | harcelement-moral-quelle-est-letendue-de-la-protection-des-victimes-et-des-temoins-du-harcelement-moral
FIXED | Qui dois-je consulter en cas de harcèlement moral? | le-harcelement-moral-quelles-sanctions-a-lencontre-de-lauteur-de-harcelement-moral | harcelement-moral-quelles-sanctions-a-lencontre-de-lauteur-de-harcelement-moral
FIXED | Qui dois-je consulter en cas de harcèlement moral? | le-harcelement-moral-quels-recours | harcelement-moral-quels-recours
FIXED | Qui dois-je consulter en cas de harcèlement moral? | le-harcelement-moral-qui-organise-la-prevention-en-matiere-de-harcelement-moral | harcelement-moral-qui-organise-la-prevention-en-matiere-de-harcelement-moral
FIXED | remise de mes documents de fin de contrat | comment-le-cdi-peut-il-prendre-fin | cdi-ce-que-dit-le-code-du-travail-comment-le-cdi-peut-il-prendre-fin
FIXED | remise de mes documents de fin de contrat | la-demission-quelle-est-la-situation-du-salarie-a-la-fin-du-contrat | demission-quelle-est-la-situation-du-salarie-a-la-fin-du-contrat
FIXED | remise de mes documents de fin de contrat | rupture-de-contrat/la-fin-de-contrat-cdd-ctt | prise-dacte-de-la-rupture-du-contrat-de-travail
FIXED | rendez vous médicaux grossesse | article-l1225-16 | l1225-16
FIXED | rupture conventionnelle collective | la-rupture-conventionnelle-collective-comment-la-rupture-conventionnelle-collective-est-elle-mise-en-oeuvre | rupture-conventionnelle-collective-comment-la-rupture-conventionnelle-collective-est-elle-mise-en-oeuvre
FIXED | rupture conventionnelle collective | la-rupture-conventionnelle-collective-laccord-de-rupture-conventionnelle-collective-peut-il-etre-conteste | rupture-conventionnelle-collective-laccord-de-rupture-conventionnelle-collective-peut-il-etre-conteste
FIXED | rupture conventionnelle collective | la-rupture-conventionnelle-collective-lobligation-de-revitalisation-des-territoires-sapplique-t-elle-dans-le-cadre-des-accords-portant-rupture-conventionnelle-collective | rupture-conventionnelle-collective-lobligation-de-revitalisation-des-territoires-sapplique-t-elle-dans-le-cadre-des-accords-portant-rupture-conventionnelle-collective
FIXED | rupture conventionnelle collective | la-rupture-conventionnelle-collective-quel-est-le-suivi-de-mise-en-oeuvre-de-laccord-de-rupture-conventionnelle | rupture-conventionnelle-collective-quel-est-le-suivi-de-mise-en-oeuvre-de-laccord-de-rupture-conventionnelle
FIXED | rupture-assistance-maternelle | les-aides-a-lemploi-dun-e-assistant-e-maternel-le-quest-ce-que-le-complement-de-libre-choix-du-mode-de-garde | assistant-e-maternel-le-quest-ce-que-le-complement-de-libre-choix-du-mode-de-garde
FIXED | rupture-assistance-maternelle | rupture-de-contrat | prise-dacte-de-la-rupture-du-contrat-de-travail
FIXED | rupture-cdd | cdd-dans-quelles-conditions-le-contrat-peut-il-etre-rompu | cdd-ce-que-dit-le-code-du-travail-dans-quelles-conditions-le-contrat-peut-il-etre-rompu
FIXED | rupture-cdd | cdd-dans-quels-cas-le-contrat-a-duree-determinee-peut-il-etre-requalifie-en-contrat-a-duree-indeterminee | cdd-ce-que-dit-le-code-du-travail-dans-quels-cas-le-contrat-a-duree-determinee-peut-il-etre-requalifie-en-contrat-a-duree-indeterminee
FIXED | rupture-cdd | cdd-lembauche-sous-contrat-a-duree-determinee | contrat-a-duree-determinee-cdd
FIXED | rupture-cdd | le-contrat-a-duree-determinee-a-objet-defini-quelle-est-sa-duree | cdd-a-objet-defini-ou-cdd-de-mission-quelle-est-sa-duree
FIXED | rupture-cdd | le-contrat-a-duree-determinee-a-objet-defini-quels-sont-la-forme-et-le-contenu-du-contrat | cdd-a-objet-defini-ou-cdd-de-mission-quels-sont-la-forme-et-le-contenu-du-contrat
FIXED | rupture-contrat-apprentissage | rupture-de-contrat | prise-dacte-de-la-rupture-du-contrat-de-travail
FIXED | rupture-contrat-professionnalisation | le-contrat-de-professionnalisation-quelles-sont-les-obligations-respectives-des-parties-au-contrat-de-professionnalisation | contrat-de-professionnalisation-alternance-quelles-sont-les-obligations-respectives-des-parties-au-contrat-de-professionnalisation
FIXED | rupture-conventionnelle | la-rupture-conventionnelle-collective-comment-est-mise-en-place-la-rupture-conventionnelle-collective | rupture-conventionnelle-collective-comment-est-mise-en-place-la-rupture-conventionnelle-collective
FIXED | rupture-conventionnelle | la-rupture-conventionnelle-collective-comment-la-rupture-conventionnelle-collective-est-elle-mise-en-oeuvre | rupture-conventionnelle-collective-comment-la-rupture-conventionnelle-collective-est-elle-mise-en-oeuvre
FIXED | rupture-conventionnelle | la-rupture-conventionnelle-collective-laccord-de-rupture-conventionnelle-collective-peut-il-etre-conteste | rupture-conventionnelle-collective-laccord-de-rupture-conventionnelle-collective-peut-il-etre-conteste
FIXED | rupture-conventionnelle | la-rupture-conventionnelle-collective-lobligation-de-revitalisation-des-territoires-sapplique-t-elle-dans-le-cadre-des-accords-portant-rupture-conventionnelle-collective | rupture-conventionnelle-collective-lobligation-de-revitalisation-des-territoires-sapplique-t-elle-dans-le-cadre-des-accords-portant-rupture-conventionnelle-collective
FIXED | rupture-conventionnelle | la-rupture-conventionnelle-du-contrat-de-travail-a-duree-indeterminee-quel-est-le-contenu-de-la-convention | rupture-conventionnelle-quel-est-le-contenu-de-la-convention
FIXED | rupture-conventionnelle | rupture-de-contrat | prise-dacte-de-la-rupture-du-contrat-de-travail
FIXED | rupture-conventionnelle-refus | la-rupture-conventionnelle-collective-comment-est-mise-en-place-la-rupture-conventionnelle-collective | rupture-conventionnelle-collective-comment-est-mise-en-place-la-rupture-conventionnelle-collective
FIXED | rupture-conventionnelle-refus | la-rupture-conventionnelle-collective-comment-la-rupture-conventionnelle-collective-est-elle-mise-en-oeuvre | rupture-conventionnelle-collective-comment-la-rupture-conventionnelle-collective-est-elle-mise-en-oeuvre
FIXED | rupture-periode-dessai | la-periode-dessai-peut-elle-etre-rompue | periode-dessai-ouvrie-la-periode-dessai-peut-elle-etre-rompue
FIXED | rupture-periode-dessai | la-periode-dessai-quelle-est-la-duree-de-la-periode-dessai | periode-dessai-ouvrie-quelle-est-la-duree-de-la-periode-dessai
FIXED | rupture-periode-dessai | le-contrat-de-professionnalisation-quelles-sont-les-modalites-de-rupture-du-contrat-de-professionnalisation-et-quelles-sont-les-demarches-a-effectuer | contrat-de-professionnalisation-alternance-quelles-sont-les-modalites-de-rupture-du-contrat-de-professionnalisation-et-quelles-sont-les-demarches-a-effectuer
FIXED | rupture-periode-dessai | rupture-de-contrat | prise-dacte-de-la-rupture-du-contrat-de-travail
FIXED | solde de tout compte | a-quel-moment-mon-employeur-doit-il-me-fournir-mon-solde-tout-compte-jq-udksinsdgfq | a-quel-moment-mon-employeur-doit-il-me-fournir-mon-solde-tout-compte-sxy2g7jtti3h5a
FIXED | solde de tout compte | les-documents-remis-aux-salaries-lors-de-la-rupture-du-contrat-de-travail-en-quoi-consiste-le-solde-de-tout-compte | rupture-du-contrat-de-travail-en-quoi-consiste-le-solde-de-tout-compte
FIXED | temps-de-travail | la-duree-legale-du-travail-quels-sont-les-temps-de-travail-comptabilises-dans-la-duree-legale | duree-legale-du-travail-quels-sont-les-temps-de-travail-comptabilises-dans-la-duree-legale
FIXED | temps-de-travail | laccord-de-performance-collective-peut-il-modifier-un-dispositif-de-damenagement-du-temps-de-travail-deja-en-place-7wdngjut0gti4g | laccord-de-performance-collective-peut-il-modifier-un-dispositif-de-damenagement-du-temps-de-travail-deja-en-place-cjrnuh4mlvkinq
FIXED | tickets-restaurant | la-remuneration-du-salarie-en-contrat-a-duree-determinee-le-salarie-peut-il-beneficier-dautres-avantages | remuneration-du-salarie-en-contrat-a-duree-determinee-le-salarie-peut-il-beneficier-dautres-avantages
FIXED | travail-dimanche | travail-du-dimanche-que-se-passe-t-il-en-cas-dinfraction | contrat-starter-que-se-passe-t-il-en-cas-de-rupture-dun-contrat-starter
FIXED | travail-dimanche | travail-du-dimanche-quelles-sont-les-derogations-accordees-par-le-prefet-ou-par-le-maire | travail-du-dimanche-iii-quelles-sont-les-derogations-accordees-par-le-prefet-ou-par-le-maire
FIXED | travail-illegal | quest-ce-que-le-travail-illegal-connaitre-la-reglementation | quest-ce-que-la-regularisation-dun-etranger-par-le-travail
FIXED | travail-nuit | le-travail-de-nuit-comment-est-mis-en-place-le-travail-de-nuit | travail-de-nuit-comment-est-mis-en-place-le-travail-de-nuit
FIXED | travail-nuit | le-travail-de-nuit-de-quel-suivi-de-son-etat-de-sante-le-travailleur-de-nuit-beneficie-t-il | travail-de-nuit-de-quel-suivi-de-son-etat-de-sante-le-travailleur-de-nuit-beneficie-t-il
FIXED | travail-nuit | le-travail-de-nuit-le-travailleur-de-nuit-quelle-definition | travail-de-nuit-le-travailleur-de-nuit-quelle-definition
FIXED | travail-nuit | le-travail-de-nuit-quelle-definition | travail-de-nuit-le-travail-de-nuit-quelle-definition
FIXED | travail-nuit | le-travail-de-nuit-quelles-sont-les-contreparties-et-les-garanties-accordees-aux-travailleurs-de-nuit | travail-de-nuit-quelles-sont-les-contreparties-et-les-garanties-accordees-aux-travailleurs-de-nuit
FIXED | valorisation des acquis de l'expérience | comment-monter-un-dossier-de-demande-de-vae | vae-validation-des-acquis-de-lexperience-comment-monter-un-dossier-de-demande-de-vae
FIXED | valorisation des acquis de l'expérience | formation | 23-formation
FIXED | valorisation des acquis de l'expérience | quelle-certification-peut-on-obtenir-avec-la-vae | vae-validation-des-acquis-de-lexperience-quelle-certification-peut-on-obtenir-avec-la-vae
FIXED | valorisation des acquis de l'expérience | qui-peut-beneficier-de-la-vae | la-vae-qui-peut-en-beneficier
FIXED | valorisation des acquis de l'expérience | vae-qui-contacter | vae-validation-des-acquis-de-lexperience-qui-contacter

julien commented:
@lionelB pas de soucis.
Très interessant votre projet. 👍

Commit:
allow 70% ES relevance

lionelB commented:
@revolunet can we merge it and add the flag to bypass datafiller in another PR (soon)

revolunet commented:
yes please :)
